README.md

```diff
@@ -10,6 +10,12 @@ Strimzi OAuth for Apache Kafka
 [Apache Kafka®](https://kafka.apache.org) comes with basic OAuth2 support in the form of SASL based authentication module which provides client-server retrieval, exchange and validation of access token used as credentials.
 For real world usage, extensions have to be provided in the form of JAAS callback handlers which is what Strimzi Kafka OAuth does.
 
+### Architecture support
+#### Linux on IBM Z (s390x) :
```

Comment:
* This basically suggests that the OAuth library supports only s390x, which is not the case. So we would need to rework the title and come up with something clearer. Or add the information about the other _supported_ architectures as well. But given this is pure Java, I think speaking about supported architectures does not much sense.
* Also, the way you added this essentially nests the sentence on line 19 under the `Linux on IBM Z (s390x)` chapter. That makes no sense. You would need to add it below it, have another title, or move this to its own section somewhere around the Building and Installing chapters, where it would make more sense I guess.